Cauli/CauliViewController/CauliViewController.swift

```diff
@@ -71,7 +72,15 @@ internal class CauliViewController: UITableViewController {
     }
 
     private func tableView(_ tableView: UITableView, detailCellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "Cell", for: indexPath)
+        let cell: UITableViewCell
+        if let description = displayingFlorets[indexPath.row].description {
+            cell = UITableViewCell(style: .subtitle, reuseIdentifier: "SubtitleCell")
```

Comment:
I think we should still dequeue a cell here instead of creating a new one.

Reply:
Done